material-ui/.stackblitzrc

```diff
@@ -0,0 +1,3 @@
+{
+  "startCommand": "npm run dev"
+}
```

Comment:
This file is not in our template. Is it needed?

Reply:
Since we're only having CodeSandbox files, I would remove it.

We can add these files for all examples in a separate PR if we want to.